website/client-old/js/controllers/inventoryCtrl.js

```diff
@@ -383,21 +382,19 @@ habitrpg.controller("InventoryCtrl",
     };
 
     $scope.shouldShowPremiumMountRow = function (potion) {
-      potion = Content.premiumHatchingPotions[potion].key;
       var pets = Object.keys(user.items.pets);
-      return pets.find(function (pet) {
-        return pet.indexOf(potion) !== -1;
+      return pets.find(function (petKey) {
```

Comment:
also, all the `shouldShowPremiumMountRow` does now is call this new function, should I remove it and use the new function directly in the mounts.jade? 
And if that's the case it needs to be `$scope.newFunction` right?